src/test/java/answers/RestAssuredAnswers1Test.java

```diff
@@ -9,7 +9,7 @@
 import io.restassured.specification.*;
 import org.junit.*;
 
-public class RestAssuredAnswers1 {
+public class RestAssuredAnswers1Test {
```

Comment:
That makes sense... I removed the `Test` suffixes in this latest release, but I knew there was a reason for me adding them initially. Now I remember :)

Obviously, the same goes for all the other name changes, highly appreciated.